### PR TITLE
travis: enable CI for MySQL 8.0 again

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
@komainu8 This PR is aimed to enable CI for MySQL 8.0 only. I'll create another PR to fix test failures.

File: tools/travis/before_script.sh [hunk 1]
```diff
@@ -70,11 +70,13 @@ else
                       "--with-mysql-build=$PWD/vendor/mysql/release")
       ;;
     mysql-8.0)
-      boost_archive=boost_1_69_0.tar.gz
-      curl -L -O https://dl.bintray.com/boostorg/release/1.69.0/source/${boost_archive}
+      #boost_archive=boost_1_69_0.tar.gz
```
Comment (komainu8) [inline tools/travis/before_script.sh]: Is this comment need?

File: tools/travis/before_script.sh [hunk 2]
```diff
@@ -70,11 +70,13 @@ else
...
+      #curl -L -O https://dl.bintray.com/boostorg/release/1.69.0/source/${boost_archive}
```
Comment (komainu8) [inline tools/travis/before_script.sh]: Ditto.

File: tools/travis/before_script.sh [hunk 3]
```diff
@@ -70,11 +70,13 @@ else
...
       sudo mkdir -p /usr/global/share
-      sudo mv ${boost_archive} /usr/global/share/
+      #sudo mv ${boost_archive} /usr/global/share/
```
Comment (komainu8) [inline tools/travis/before_script.sh]: Ditto.

File: tools/travis/before_script.sh (hunk 1 above)
Comment (kenhys) [reply tools/travis/before_script.sh]: At first, the download issue will be fixed in upstream, then I think to get them back.(so I've treated them as comments)
Now, it seems better to remove old stuffs. so I'll remove them.

File: tools/travis/before_script.sh (hunk 2 above)
Comment (kenhys) [reply tools/travis/before_script.sh]: removed.

File: tools/travis/before_script.sh (hunk 3 above)
Comment (kenhys) [reply tools/travis/before_script.sh]: removed

**Comment by kenhys** [comment]
@komainu8 I've fixed commits. Thanks!

**Pull request merged**